lib.rs

```diff
@@ -22,62 +18,41 @@ mod chocolate {
         reviews: Mapping<u32, Review>,
         /// Accountid + projectId
         reviews_projects_list: Vec<(AccountId, u32)>,
+        /// Stores the address of account that have initiated the verification flow.
+        account_verification_flow_initiation: Mapping<AccountId, bool>, // TODO: Maybe use a bool?
```

Comment:
This should map to the message we send out, then we can verify that it's the right signature we got back

Reply:
Okay, the storage for AccountId in ink 4.0 has been changed. I'm still figuring it out. Thanks for the heads up.